Build failure with latest LLVM

Here's the build error, using LLVM from earlier today (revision r304435):

```
[punpacking sources
[3punpacking source archive /nix/store/rh5yc6dpy1r6j1nlrcjp7j6jrmcqks8b-dwarf-type-reader-93e27ce051dffa78b5eec92545c052f3a5e27f19-src
[qsource root is dwarf-type-reader-93e27ce051dffa78b5eec92545c052f3a5e27f19-src
[q[ppatching sources
[q[pconfiguring
fixing cmake files...
cmake flags: -DCMAKE_BUILD_TYPE=Release -DCMAKE_SKIP_BUILD_RPATH=ON -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/vc252ypgwdnxnv7zafqrcdq1kkmbwbg9-dwarf-type-reader-2017-05-04/include -DCMAKE_INSTALL_LIBDIR=/nix/store/vc252ypgwdnxnv7zafqrcdq1kkmbwbg9-dwarf-type-reader-2017-05-04/lib -DCMAKE_INSTALL_NAME_DIR=/nix/store/vc252ypgwdnxnv7zafqrcdq1kkmbwbg9-dwarf-type-reader-2017-05-04/lib -DCMAKE_INSTALL_PREFIX=/nix/store/vc252ypgwdnxnv7zafqrcdq1kkmbwbg9-dwarf-type-reader-2017-05-04 -DLLVM_ROOT=/nix/store/hscbf5g6wqd12gamqzycihannznkgg2f-llvm-5.0.0-304435 -DLLVM_BUILD_TOOLS=ON 
-- The C compiler identification is GNU 5.4.0
-- The CXX compiler identification is GNU 5.4.0
-- Check for working C compiler: /nix/store/d7c948x5a81rpl36b3aypgc3wn1javgi-gcc-wrapper-5.4.0/bin/gcc
-- Check for working C compiler: /nix/store/d7c948x5a81rpl36b3aypgc3wn1javgi-gcc-wrapper-5.4.0/bin/gcc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /nix/store/d7c948x5a81rpl36b3aypgc3wn1javgi-gcc-wrapper-5.4.0/bin/g++
-- Check for working CXX compiler: /nix/store/d7c948x5a81rpl36b3aypgc3wn1javgi-gcc-wrapper-5.4.0/bin/g++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test C_SUPPORTS_FPIC
-- Performing Test C_SUPPORTS_FPIC - Success
-- Performing Test CXX_SUPPORTS_FPIC
-- Performing Test CXX_SUPPORTS_FPIC - Success
-- Building with -fPIC
-- Performing Test SUPPORTS_FVISIBILITY_INLINES_HIDDEN_FLAG
-- Performing Test SUPPORTS_FVISIBILITY_INLINES_HIDDEN_FLAG - Success
-- Performing Test C_SUPPORTS_WERROR_DATE_TIME
-- Performing Test C_SUPPORTS_WERROR_DATE_TIME - Success
-- Performing Test CXX_SUPPORTS_WERROR_DATE_TIME
-- Performing Test CXX_SUPPORTS_WERROR_DATE_TIME - Success
-- Performing Test CXX_SUPPORTS_CXX11
-- Performing Test CXX_SUPPORTS_CXX11 - Success
-- Performing Test C_SUPPORTS_FNO_FUNCTION_SECTIONS
-- Performing Test C_SUPPORTS_FNO_FUNCTION_SECTIONS - Success
-- Performing Test C_SUPPORTS_FFUNCTION_SECTIONS
-- Performing Test C_SUPPORTS_FFUNCTION_SECTIONS - Success
-- Performing Test CXX_SUPPORTS_FFUNCTION_SECTIONS
-- Performing Test CXX_SUPPORTS_FFUNCTION_SECTIONS - Success
-- Performing Test C_SUPPORTS_FDATA_SECTIONS
-- Performing Test C_SUPPORTS_FDATA_SECTIONS - Success
-- Performing Test CXX_SUPPORTS_FDATA_SECTIONS
-- Performing Test CXX_SUPPORTS_FDATA_SECTIONS - Success
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_INSTALL_INCLUDEDIR
    CMAKE_INSTALL_LIBDIR


-- Build files have been written to: /tmp/nix-build-dwarf-type-reader-2017-05-04.drv-0/dwarf-type-reader-93e27ce051dffa78b5eec92545c052f3a5e27f19-src/build
[q[pbuilding
make flags: SHELL=/nix/store/jn022hrd3cxflcqmkvmq4mqr07sws08v-bash-4.3-p46/bin/bash    
Scanning dependencies of target utils
[ 40%] Building CXX object lib/CMakeFiles/utils.dir/utils.cpp.o
[ 40%] Building CXX object lib/CMakeFiles/utils.dir/variable_type.pb.cc.o
In file included from /nix/store/kpm2p6qjhvhl6km26vg07l727hgna6rc-gcc-5.4.0/include/c++/5.4.0/cassert:43:0,
                 from /nix/store/hscbf5g6wqd12gamqzycihannznkgg2f-llvm-5.0.0-304435/include/llvm/DebugInfo/DWARF/DWARFDie.h:22,
                 from /tmp/nix-build-dwarf-type-reader-2017-05-04.drv-0/dwarf-type-reader-93e27ce051dffa78b5eec92545c052f3a5e27f19-src/lib/utils.h:6,
                 from /tmp/nix-build-dwarf-type-reader-2017-05-04.drv-0/dwarf-type-reader-93e27ce051dffa78b5eec92545c052f3a5e27f19-src/lib/utils.cpp:1:
/tmp/nix-build-dwarf-type-reader-2017-05-04.drv-0/dwarf-type-reader-93e27ce051dffa78b5eec92545c052f3a5e27f19-src/lib/utils.cpp: In member function 'void DwarfVariableFinder::findVariablesInScope(const llvm::DWARFDie&)':
/tmp/nix-build-dwarf-type-reader-2017-05-04.drv-0/dwarf-type-reader-93e27ce051dffa78b5eec92545c052f3a5e27f19-src/lib/utils.cpp:97:53: error: no matching function for call to 'llvm::DWARFDie::getLowAndHighPC(uint64_t&, uint64_t&) const'
       assert(scope_die.getLowAndHighPC(LowPC, HighPC));
                                                     ^
In file included from /tmp/nix-build-dwarf-type-reader-2017-05-04.drv-0/dwarf-type-reader-93e27ce051dffa78b5eec92545c052f3a5e27f19-src/lib/utils.h:6:0,
                 from /tmp/nix-build-dwarf-type-reader-2017-05-04.drv-0/dwarf-type-reader-93e27ce051dffa78b5eec92545c052f3a5e27f19-src/lib/utils.cpp:1:
/nix/store/hscbf5g6wqd12gamqzycihannznkgg2f-llvm-5.0.0-304435/include/llvm/DebugInfo/DWARF/DWARFDie.h:198:8: note: candidate: bool llvm::DWARFDie::getLowAndHighPC(uint64_t&, uint64_t&, uint64_t&) const
   bool getLowAndHighPC(uint64_t &LowPC, uint64_t &HighPC,
        ^
/nix/store/hscbf5g6wqd12gamqzycihannznkgg2f-llvm-5.0.0-304435/include/llvm/DebugInfo/DWARF/DWARFDie.h:198:8: note:   candidate expects 3 arguments, 2 provided
lib/CMakeFiles/utils.dir/build.make:62: recipe for target 'lib/CMakeFiles/utils.dir/utils.cpp.o' failed
make[2]: *** [lib/CMakeFiles/utils.dir/utils.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
CMakeFiles/Makefile2:218: recipe for target 'lib/CMakeFiles/utils.dir/all' failed
make[1]: *** [lib/CMakeFiles/utils.dir/all] Error 2
Makefile:127: recipe for target 'all' failed
make: *** [all] Error 2
[q
```

I'll probably pin to a known-working revision but thought I'd let you know.

(And you might want to mention what revision is supported/known-working in the README or something)

--- a/lib/utils.cpp
+++ b/lib/utils.cpp
@@ -92,9 +92,9 @@ void DwarfVariableFinder::findVariablesInScope(const DWARFDie &scope_die) {
       scope->set_symbol_name(
           dwarf::toString(scope_die.find(dwarf::DW_AT_name), "None"));
 
-      uint64_t LowPC, HighPC;
+      uint64_t LowPC, HighPC, SectionIdx;
       LowPC = HighPC = ~0U;
-      assert(scope_die.getLowAndHighPC(LowPC, HighPC));
+      assert(scope_die.getLowAndHighPC(LowPC, HighPC, SectionIdx));
       scope->set_entry_address(LowPC);
 
       auto *var = LV->mutable_var();

